test/NuGet.Core.Tests/NuGet.Build.Tasks.Test/WarnForInvalidProjectsTaskTests.cs

```diff
@@ -131,7 +128,6 @@ public void WarnForInvalidProjectsTask_AllValidItemsVerifyNoWarnings()
 
             testLogger.Warnings.Should().Be(0);
             testLogger.Errors.Should().Be(0);
-            testLogger.DebugMessages.Count.Should().Be(2);
```

Comment:
Should these be 0 instead of removing them completely?

Reply:
I don't think it matters unless we really care to ensure that a debug message isn't accidentally added.  The code diff clearly shows that I removed the existing ones so it seems odd to me to check that they were removed.  Do you feel strongly that we should update the test instead of remove this assertion?

Reply:
it's one of those that if I was writing the code originally, I would've changed it to 0. 

I don't really have a strong reason to add it back though. I'm ok with it as is.